.storybook/main.js

```diff
@@ -11,6 +11,17 @@ module.exports = {
     '@storybook/addon-storysource',
     // '@storybook/addon-knobs',
     'storybook-dark-mode',
+    {
+      name: '@storybook/addon-postcss',
+      options: {
+        postcssLoaderOptions: {
+          postcssOptions: {
+            config: './.storybook/postcss.config.js',
```

Comment:
[IMO] そうか、この path ってプロジェクトルートからの相対なんですね。

`path.resolve` とかで書いたほうがその辺意識しないコードにできるかも

```suggestion
            config: path.resolve(__dirname, 'postcss.config.js'),
```

Reply:
たしかに…
`__dirname` 使わないにしても OS の差異吸収を考えると `path.resolve` は使ったほうが良いですね、変えます